3-binary-data/lib/images.js

```diff
@@ -67,16 +67,14 @@ function sendUploadToGCS (req, res, next) {
 // [END process]
 
 // Multer handles parsing multipart/form-data requests.
-// This instance is configured to store images in memory and re-name to avoid
-// conflicting with existing objects. This makes it straightforward to upload
-// to Cloud Storage.
+// This instance is configured to store images in memory.
+// This makes it straightforward to upload to Cloud Storage.
 // [START multer]
-var multer = require('multer')({
-  inMemory: true,
-  fileSize: 5 * 1024 * 1024, // no larger than 5mb
-  rename: function (fieldname, filename) {
-    // generate a unique filename
-    return filename.replace(/\W+/g, '-').toLowerCase() + Date.now();
+var Multer = require('multer');
+var multer = Multer({
```

Comment:
Even though I need to reference `Multer` on the line below, for `Multer.MemoryStorage`? (Though `MemoryStorage` may be the default option, and thus omittable - if that's the case, I'm happy to change this.)